ROADMAP.md

```diff
@@ -78,3 +90,27 @@
 
  * [ ] Tracing integration for advanced performance measurements, see google chrome chrome://tracing/ and flame graphs
        See lttng, add trace points to the source code on the important functions
+
+## Quality Of Life Improvements
+
+ * [ ] Evaluate and refactor basic error handling approach based on enums
+    * all error classes should implement `std::error::Error` for the non `no_std` build  (maybe use `thiserror`)
+    * should be compatible with `anyhow` and `eyre`
+    * explore error pyramid concept
+ * [ ] Evaluate crate `log` and `tracing` as backend for elkodon logger
+    * `log` / `tracing` / `console_logger` one of them should be the default logger, depending on feature flagA
+ * [ ] Use `&str` and UTF-8 in `ServiceName`, there is no need for a length or ASCII restriction
+ * [ ] Rename `enable_safe_overflow` into `set_safe_overflow` in `ServiceBuilder` `publish_subscribe`
+    * or maybe rename it into behavior: queue and ringbuffer, get inspired by crossbeam queues
+ * [ ] Rename `publisher::loan` into `publisher::loan_uninit` and provide `publisher::loan` with default
+    constructed type
+ * [ ] Provide `[T]` (slice) as special transmission type for pub/sub
+    * `loan` and `loan_uninit` with alignment and number of Ts
```

Comment:
I guess this needs to become `loan_slice`, `loan_uninit_slice` and `loan_uninit_slice_with_alignment`